src/hooks/screen/useChatScreenHook.ts

```diff
@@ -23,6 +24,7 @@ import {QueueJobPriority} from '../../core/queue/BaseQueue';
 import {getAnonymousUserId, getUserId} from '../../utils/users';
 import {randomString} from '../../utils/string/StringUtils';
 import {useGetAllMessage} from './services/chatScreenHooks';
+import {useSendAnonMessage, useSendSignedMessage} from '../../service/chat';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useSendAnonMessage` and `useSendSignedMessage` from '../../service/chat' is added. These are custom hooks that encapsulate the logic for sending anonymous and signed messages respectively.
```diff
+ import {useSendAnonMessage, useSendSignedMessage} from '../../service/chat';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->